codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectAggregator.java

```diff
@@ -188,12 +188,35 @@ public void operationComplete(ChannelFuture future) throws Exception {
         }
     }
 
-    private abstract static class AggregatedFullHttpMessage implements ByteBufHolder, FullHttpMessage {
+    protected AggregatedHttpRequest createAggregatedHttpRequest(HttpRequest request,
```

Comment:
we usually use "newAggregateHttpRequest" etc. Can you please replace "create" by "new" everywhere ?